### PR TITLE
[Development] Update yaml files

### PR DESCRIPTION
**Pull request by Zhouxunzhe**
## Motivation and Context
Update yaml files for new features: single agent, place action, nav sensor

## How Has This Been Tested
**Tested `single agent` on spot, fetch, stretch, tested `place action` and `nav sensor` only on spot robot.**

## Types of changes
\[Development\]

- **\[Docs change\]** Addition or changes to the documentation
- **\[Refactoring\]** Large changes to the code that improve its functionality or performance
- **\[Dependency Upgrade\]** Upgrades one or several dependencies in habitat
- **\[Bug Fix\]** (non-breaking change which fixes an issue)
- **\[Development\]** A pull request that add new features to the [habitat-lab](/habitat-lab) task and environment codebase. Development Pull Requests must be small (less that 500 lines of code change), have unit testing, very extensive documentation and examples. These are typically new tasks, environments, sensors, etc... The review process for these Pull Request is longer because these changes will be maintained by our core team of developers, so make sure your changes are easy to understand!
- **\[Experiment\]** A pull request that add new features to the [habitat-baselines](/habitat-baselines/) training codebase. Experiments Pull Requests can be any size, must have smoke/integration tests and be isolated from the rest of the code. Your code additions should not rely on other habitat-baselines code. This is to avoid dependencies between different parts of habitat-baselines. You must also include a README that will document how to use your new feature or trainer. **You** will be the maintainer of this code, if the code becomes stale or is not supported often enough, we will eventually remove it.

## Checklist

- [x] My code follows the code style of this project.
- [x] I have updated the documentation if required.
- [x] I have read the [**CONTRIBUTING**](/CONTRIBUTING.md) document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [x] I have added tests to cover my changes if required.

### CODE REVIEW

File: habitat-lab/habitat/config/habitat/simulator/agents/habitat_mas_agents/SpotRobot_default.yaml [hunk 1]
```diff
@@ -8,11 +8,11 @@ defaults:
...
-  - /habitat/simulator/sim_sensors@sim_sensors.head_stereo_left_depth_sensor: head_stereo_left_depth_sensor
-  - /habitat/simulator/sim_sensors@sim_sensors.head_stereo_right_depth_sensor: head_stereo_right_depth_sensor
-  - /habitat/simulator/sim_sensors@sim_sensors.jaw_rgb_sensor: jaw_rgb_sensor
-  - /habitat/simulator/sim_sensors@sim_sensors.jaw_depth_sensor: jaw_depth_sensor
-  - /habitat/simulator/sim_sensors@sim_sensors.jaw_semantic_sensor: jaw_semantic_sensor
+  #- /habitat/simulator/sim_sensors@sim_sensors.head_stereo_left_depth_sensor: head_stereo_left_depth_sensor
+  #- /habitat/simulator/sim_sensors@sim_sensors.head_stereo_right_depth_sensor: head_stereo_right_depth_sensor
+  #- /habitat/simulator/sim_sensors@sim_sensors.jaw_rgb_sensor: jaw_rgb_sensor
+  #- /habitat/simulator/sim_sensors@sim_sensors.jaw_depth_sensor: jaw_depth_sensor
+  #- /habitat/simulator/sim_sensors@sim_sensors.jaw_semantic_sensor: jaw_semantic_sensor
```
Comment (SgtVincent) [inline habitat-lab/habitat/config/habitat/simulator/agents/habitat_mas_agents/SpotRobot_default.yaml:15]: Why would you comment out stereo and jaw sensors?

File: habitat-lab/habitat/config/habitat/dataset/rearrangement/zxz_manipulation.yaml [hunk 2]
```diff
@@ -6,5 +6,6 @@ defaults:
 
 type: RearrangeDataset-v0
 split: eval
-data_path: data/datasets/manipulation/manipulation_eval.json.gz
+#data_path: data/datasets/single_agent_eval.json.gz
+data_path: data/datasets/single_agent_success.json.gz
```
Comment (SgtVincent) [inline habitat-lab/habitat/config/habitat/dataset/rearrangement/zxz_manipulation.yaml]: Where are those files? I did not see them in google drive.

File: habitat-lab/habitat/config/benchmark/single_agent/pddl/zxz_move_test.yaml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - move_forward(robot_0)
+  - turn_left(robot_0)
+  - turn_right(robot_0)
```
Comment (SgtVincent) [inline habitat-lab/habitat/config/benchmark/single_agent/pddl/zxz_move_test.yaml:110]: What is this crazy file for? Can you explain a little bit? :astonished:

File: habitat-lab/habitat/config/benchmark/multi_agent/zxz_spot_drone_per.yaml [hunk 4]
```diff
@@ -135,8 +119,8 @@ habitat:
       agent_0:
         joint_start_noise: 0.0
   dataset:
-    data_path: data/datasets/perception/perception_eval.json.gz
+    data_path: data/datasets/hssd_eval.json.gz
     # new
-    randomize_agent_start: 0
+    randomize_agent_start: 1
```
Comment (SgtVincent) [inline habitat-lab/habitat/config/benchmark/multi_agent/zxz_spot_drone_per.yaml]: Why would we  want to randomize agent initial position **BY DEFAULT**?

File: habitat-baselines/habitat_baselines/config/social_rearrange/zxz_llm_spot_drone_per.yaml [hunk 5]
```diff
@@ -41,7 +41,8 @@ habitat_baselines:
   evaluator:
     _target_: habitat_baselines.rl.multi_agent.habitat_mas_evaluator.HabitatMASEvaluator
 
-  image_dir: "video_dir/image_dir/"
+  image_dir: "video_multi/spot_drone/image_dir/"
+  video_dir: "video_multi/spot_drone"
```
Comment (SgtVincent) [inline habitat-baselines/habitat_baselines/config/social_rearrange/zxz_llm_spot_drone_per.yaml]: Please disable image saving by default, it is very slow and not desired by everyone. Simply commented out this line and  add a comment for this option.

**Review by SgtVincent — CHANGES_REQUESTED** [review]
Please address my comments and let me know when you are ready for a second review.

File: habitat-lab/habitat/config/habitat/simulator/agents/habitat_mas_agents/SpotRobot_default.yaml (hunk 1 above)
Comment (Zhouxunzhe) [reply habitat-lab/habitat/config/habitat/simulator/agents/habitat_mas_agents/SpotRobot_default.yaml:15]: I was just to lower the sensors to be demonstrated for convenience. If you need it to be changed back to all sensors, please tell me.

File: habitat-lab/habitat/config/habitat/dataset/rearrangement/zxz_manipulation.yaml (hunk 2 above)
Comment (Zhouxunzhe) [reply habitat-lab/habitat/config/habitat/dataset/rearrangement/zxz_manipulation.yaml]: These are for locally test, new yamls and datasets have been uploaded on drive.

File: habitat-lab/habitat/config/benchmark/single_agent/pddl/zxz_move_test.yaml (hunk 3 above)
Comment (Zhouxunzhe) [reply habitat-lab/habitat/config/benchmark/single_agent/pddl/zxz_move_test.yaml:110]: This is aiming for locally test, sorry for uploading it. I have deleted it on Github.

File: habitat-lab/habitat/config/benchmark/multi_agent/zxz_spot_drone_per.yaml (hunk 4 above)
Comment (Zhouxunzhe) [reply habitat-lab/habitat/config/benchmark/multi_agent/zxz_spot_drone_per.yaml]: Have solved it. It is now the same as before, and the robot configs has been uploaded on drive.

File: habitat-baselines/habitat_baselines/config/social_rearrange/zxz_llm_spot_drone_per.yaml (hunk 5 above)
Comment (Zhouxunzhe) [reply habitat-baselines/habitat_baselines/config/social_rearrange/zxz_llm_spot_drone_per.yaml]: The path doesn't affect whether it saves images. It was set not to save images if we just leave `image_option` blank.
https://github.com/SgtVincent/habitat-lab/blob/2a7b9c7c433d58d5e6dc96bfe1979344edf581c5/habitat-baselines/habitat_baselines/config/social_rearrange/zxz_llm_spot_drone_per.yaml#L49-L50

File: habitat-lab/habitat/config/habitat/simulator/agents/habitat_mas_agents/SpotRobot_default.yaml (hunk 1 above)
Comment (SgtVincent) [reply habitat-lab/habitat/config/habitat/simulator/agents/habitat_mas_agents/SpotRobot_default.yaml:15]: Please add them back in the SpotRobot_default.yaml. If people want a spot robot with a different sensor setup, they should use SpotRobot_<sensor_setup>.yaml instead. This is what the config system is designed for.

**Review by SgtVincent — APPROVED** [review]
LGTM!

**Pull request merged**